frontend/src/metabase/services.js

```diff
@@ -1,745 +1,171 @@
-            method: 'POST',
-            params: {
-                dbId: '@dbId'
-            }
-        },
-        table_list: {
-            url: '/api/table/',
-            method: 'GET',
-            params: {
-                tableId: '@tableId'
-            },
-            isArray: true
-        },
-        table_get: {
-            url: '/api/table/:tableId',
-            method: 'GET',
-            params: {
-                tableId: '@tableId'
-            }
-        },
-        table_update: {
-            url: '/api/table/:tableId',
-            method: 'PUT',
-            params: {
-                tableId: '@id'
-            }
-        },
-        table_fields: {
-            url: '/api/table/:tableId/fields',
-            method: 'GET',
-            params: {
-                tableId: '@tableId'
-            },
-            isArray: true
-        },
-        table_fks: {
-            url: '/api/table/:tableId/fks',
-            method: 'GET',
-            params: {
-                tableId: '@tableId'
-            },
-            isArray: true
-        },
-        table_reorder_fields: {
-            url: '/api/table/:tableId/reorder',
-            method: 'POST',
-            params: {
-                tableId: '@tableId'
-            }
-        },
-        table_query_metadata: {
-            url: '/api/table/:tableId/query_metadata',
-            method: 'GET',
-            params: {
-                dbId: '@tableId'
-            }
-        },
-        table_sync_metadata: {
-            url: '/api/table/:tableId/sync',
-            method: 'POST',
-            params: {
-                tableId: '@tableId'
-            }
-        },
-        field_get: {
-            url: '/api/field/:fieldId',
-            method: 'GET',
-            params: {
-                fieldId: '@fieldId'
-            }
-        },
-        field_summary: {
-            url: '/api/field/:fieldId/summary',
-            method: 'GET',
-            params: {
-                fieldId: '@fieldId'
-            },
-            isArray: true
-        },
-        field_values: {
-            url: '/api/field/:fieldId/values',
-            method: 'GET',
-            params: {
-                fieldId: '@fieldId'
-            }
-        },
-        field_value_map_update: {
-            url: '/api/field/:fieldId/value_map_update',
-            method: 'POST',
-            params: {
-                fieldId: '@fieldId'
-            }
-        },
-        field_update: {
-            url: '/api/field/:fieldId',
-            method: 'PUT',
-            params: {
-                fieldId: '@id'
-            }
-        },
-        dataset: {
-            url: '/api/dataset',
-            method: 'POST',
-            then: function(resolve) {
-                // enable cancelling of the request using this technique:
-                // http://www.nesterovsky-bros.com/weblog/2015/02/02/CancelAngularjsResourceRequest.aspx
-                if (this.params) {
```

Comment:
I added basically the same thing but it's part of `metabase/lib/api` so it works on every method now without this hack.